ui/src/message.ts

```diff
@@ -57,7 +57,7 @@ export interface RoomInfo {
 export interface RoomUser {
     id: string;
     name: string;
-    sharing: boolean;
+    streaming: boolean;
```

Comment:
This interface only defines the payload created by the backend. This flag also needs to be adjusted in:

https://github.com/screego/server/blob/53a647e15881abf1099e9bc67b05a7764c5fe667/ws/outgoing/messages.go#L19-L25

Reply:
Will adjust and this time also build and test!